### ISSUE TITLE
Slow release pace 

### ISSUE THREAD

**Issue by samarulmeu** [yshui/compton#1294]
It seems that lately the release pace just slowed down. The last release is in February and it seems there are a lot a bug fixes and improvements. And also I think that issue #1207 should also be implemented as a way to check for the new ones.

**Comment by yshui** [comment]
Well, I don't know what you mean by "slowed down". v10 to v11 took more than a year. I'd say release pace has sped up.

And I need you to understand, I am not getting paid to do this, so I only have limited time and energy I can spend on this. (I guess that's not entirely true now. While I am grateful to the people who sponsor my work, realistically this hasn't made a difference to the amount of time I have available for this).

If you really want to see a faster development pace, well, contribute.

**Comment by samarulmeu** [comment]
Maybe my message was not understood the way I intended. I truly appreciate all your work, as I appreciate any work in the OpenSource community, and when I can I contribute. 

My message was about the intermediary releases like. 11.1, 11.2. Maybe there could have also been a 11.3 with some of the fixed bugs. I understand that major releases like 11, 12, 13 will take time. 

Anyhow my message was not intended to be a reproach and hope no harm was done. In the end is your software, your law. And I wrote as a regular user. In OpenBSD and Devuan I have to compile the releases as the ones available in the repositories are eating all the RAM and are unusable. Intermediary release means that I can check when something new or fixed is available, compile and use the new version.

Best thoughts!
